features/exploredash/src/main/res/drawable/round_corners_bgd_white.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <corners android:radius="8dp"/>
+    <solid android:color="@color/dash_white"/>
+</shape>
```

Comment:
This resource already exists in the wallet module, perhaps it's better to move it to the common instead of duplicating. Or you can use `rounded_background` with a theme, the purpose of `rounded_background` is to avoid creating multiple rounded background resources but instead use just one with different themes.